### DISCUSSION TITLE
M.I.B. not open FEC

### DISCUSSION THREAD

**Discussion by uds123** [Mr-MIBonk/M.I.B._More-Incredible-Bash#286] [Q&A]
Hi, need to ask your help, unfortunately can to do it by myself! 
I have Q7 2018 4M with MHI2Q_ER_AUG22_P5092 1392 version. I install latest MIB to access GEM. I can access GEM, but cant add any of FEC to car, for example - on menu is now visible Audi Smartphone interface, but I cant connect ay phone. As well I try to add map FEC , but its not working, all the time shows import fail.
In the same time I can import speed camera POI in to map.

Thank you in advance!

**Comment by uds123** [comment]
Can it means that existing PATCHs are not working? I try to uninstall and install again but result is the same. Its seems its not importing FECs.

**Reply by harman-f** [reply]
Did you run AIO patch?

Just adding FECs will not work!

Share your LOG file

**Reply by uds123** [reply]
Hi, 
please see:
[MU1329-JTB-07425.10.18200106CD.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/12138354/MU1329-JTB-07425.10.18200106CD.log)

**Reply by uds123** [reply]
Yes I run AIO patch as well.

**Reply by harman-f** [reply]
`Patch folder /net/mmx/fs/sda0/patches/MHI2Q_ER_AUG22_P5092_MU1329_PATCH NOT found on SD card`

Well, that LOG looks interesting...

copy patch file to SD and patch 1st!

If that worked you can continue with enabling navigation and CP/AA.

**Reply by harman-f** [reply]
Use latest M.I.B: https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/archive/refs/heads/main.zip

**Comment by uds123** [comment]
Hi, thank you for advice! Its works now!
